onboarding/.gitignore

```diff
@@ -0,0 +1,14 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
Should we keep this?

Reply:
I'd keep the file for now, and can do a subsequent PR that moves this and the prettier config up one directory to make it work for all the other samples.

Reply:
I meant the comment, we can keep the file.